src/main/cpp/include/loader/tiledb_loader_text_file.h

```diff
@@ -175,7 +175,7 @@ class LineBasedTextFile2TileDBBinary : public File2TileDBBinaryBase {
                             treat_deletions_as_intervals,
                             parallel_partitions, noupdates, close_file) {
     vid_mapper.build_tiledb_array_schema(m_array_schema, "dummy",
-                                         false, false);
+                                         false, 0, 0, false);
```

Comment:
Use `TILEDB_NO_COMPRESSION` or `TILEDB_GZIP` for compression and `0` and `TILEDB_COMPRESSION_LEVEL_GZIP` based on the compression default.